learn/framework/xcm.md

```diff
@@ -0,0 +1,99 @@
+- **Absolute** -  messages are guaranteed to be delivered to the intended destination, in order and in a timely fashion
+- **Asymmetric** -  messages sent have no response counterpart. Any return values, if required, must be sent back from the destination to the sender with another message
+- **Agnostic** -  there are no assumptions whatsoever about the configuration or properties of two communicating appchains. Appchains might differ in every aspect, except the ability to understand XCM. E.g., one chain could be EVM-compatible and not the other, one chain could be a DeFi appchain and the other a gaming appchain, and so on
+
+## Common Use Cases {: #common-use-cases }
+
+Many use cases can be addressed by benefiting from the common ground and versatility XCM provides. Two of the most recurrent ones are asset transfers and remote execution.
+
+### Asset Transfers {: #asset-transfer }
+
+Moving digital assets from one appchain to another is essential for creating a more dynamic, efficient, and interconnected blockchain ecosystem. The native cross-chain capability allows two main strategies to transfer assets from one chain to another:
+
+- **Teleport** - teleporting an asset is a simple and efficient mechanism, but it has a major caveat: it requires trust between the parties. In essence, when appchain A wants to send X amount of assets to appchain B, it burns X amount of assets and sends a message to appchain B instructing them to mint exactly X amount of assets, preserving the overall asset balance and concluding the teleport action
+- **Reserve transfer** - A reserve transfer is executed on the **reserve chain**, which is the chain where the asset being transferred is native (e.g., [Moonbeam](https://moonbeam.network/){target=\_blank} is the reserve chain for the GLMR token). Also, appchains hold a **sovereign account** on the reserve chain, a keyless account managed by the respective appchain governor. Thus, when appchain A wants to send X amount of an asset to appchain B, it sends the instructions to the reserve chain, which transfers the assets from appchain A to appchain B's sovereign accounts, and the instructions to appchain B to mint X amount of a derivative form of the transferred asset. Note that regardless of appchain B minting tokens in excess, since the wrongful tokens are not backed one-to-one by the reserve, they have no real value
+
+### Remote Execution {: #remote-execution }
+
+The native interoperability XCM provides allows an appchain to send a message to another triggering some action. For example, If the destination chain is EVM-compatible, appchain A can call a smart contract deployed on appchain B.
+
+To get any on-chain request executed, it is necessary to cover its associated fees. On XCM, remote execution can be bought with two steps:
```

Comment:
This is also true for token transfers, so maybe move this to the general chapter. XCM execution in  the destination chain also neds to buy execution

Reply:
Moved the fees section up, so it comes before remote transfers and execution.
(Also added a line about fees in the transfer section)